sequentialthinking-mcp-server/.actor/actor.json

```
@@ -0,0 +1,22 @@
+{
+	"actorSpecification": 1,
+	"name": "sqt-mcp-server",
+	"title": "Sequential Thinking MCP server",
+		"description": "Dynamic, step-by-step problem-solving via structured thinking. Exposes the sequential_thinking tool as an MCP server for Apify clients.",
```

```suggestion
		"description": "An MCP server for structured sequential thinking process - breaking down complex problems, iteratively refining solutions, and exploring multiple reasoning paths.",
```

nit: What about description like this?